flake.nix

```diff
@@ -505,14 +524,36 @@
 
       # https://nixos.org/manual/nixos/unstable/index.html#sec-calling-nixos-tests
       runNixOSTestFor = system: test: nixos-lib.runTest {
-        imports = [ test ];
+        imports = [
+          test
+          # For fast cycle when you don't need the regular tests:
+          # (useNixWithoutRegularTests system)
```

Comment:
Not enabled by default because it does duplicate the compilation work if you want to run both VM and non-VM tests.